pkg/cli/mirror/copy.go

```diff
@@ -741,14 +797,14 @@ func pullImage(from, to string, srcSkipTLS bool, inStyle style, funcs RemoteRegF
 
 // pushImage is here used only to push images to the remote registry
 // calls the underlying containers/image copy library
-func pushImage(from, to string, dstSkipTLS bool, insecurePolicy bool, funcs RemoteRegFuncs) error {
+func pushImage(from, to string, dstSkipTLS bool, insecurePolicy bool, funcs RemoteRegFuncs) (digest.Digest, error) {
 
 	// find absolute path if from is a relative path
 	fromPath := trimProtocol(from)
 	if !strings.HasPrefix(fromPath, "/") {
 		absolutePath, err := filepath.Abs(fromPath)
 		if err != nil {
-			return fmt.Errorf("unable to get absolute path of oci image %s: %v", from, err)
+			return "", fmt.Errorf("unable to get absolute path of oci image %s: %v", from, err)
```

Comment:
nit:  really a matter of preference but would this look better (from a maintenance/dev perspective) 

`digest.Digest("") `